packaging/linux/linux_agent_builder.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+set -eux
+
+DISTRO="${1:-centos}"
+RELEASE="${2:-core}"
+CFY_VERSION="$3"
+
+AGT_NAME="$DISTRO-$RELEASE-agent_$CFY_VERSION"
+WORKDIR="$(pwd)"
+
+mkdir -p cloudify/env
+
+# This is going to create an agent .tar.gz with a python inside the agent's
+#   env/ folder, not a true virtualenv.
+
+# -- copy python 3.10 to "venv"
+curl https://cloudify-cicd.s3.amazonaws.com/python-build-packages/cfy-python3.10.tgz -o cfy-python3.10.tgz
+tar --strip-components=2 -zxf cfy-python3.10.tgz -C cloudify/env/
+pushd 'cloudify/env/bin'
+    ln -s python3.10 python
+    ln -s python3.10 python3
+    ln -s pip3.10 pip
+    ln -s pip3.10 pip3
+popd
+
+# -- install agent
+cloudify/env/bin/python3.10 -m pip install setuptools upgrade
+cloudify/env/bin/python3.10 -m pip install -r dev-requirements.txt
```

Comment:
Installing dev-requirements for something that is part of the (eventual) production pipeline? That seems wrong. Have we got a badly named reqs file?

Reply:
this actually installs cloudify-common, so maybe just rename the file requirements.txt ?

Reply:
Let's do the very-correct thing, with `reqs.in` and `reqs.txt` and `pip-compile`, but... not right now. I'd like to do this as part of also relaxing pins in setup.py (the same way we did in restservice etc).

So: yes, but please not now.